src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+import $style from './slider-improvements.module.css';
+import SliderButton from '@src/features/basic/slider-improvements/SliderButton.vue';
+import { clickElement } from '@src/util';
+
+function onTileReady(tile: PrunTile) {
+  sliderMinMaxButtons(tile);
+  sliderTableBulkMinMax(tile);
+}
+
+function sliderTableBulkMinMax(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    const headerRow = _$(table, 'thead')?.firstElementChild;
+    if (!headerRow) return;
+    const sliders = _$$(table, 'rc-slider');
+    if (sliders.length > 0) {
+      const sliderColumns = new Map<number, HTMLElement[]>();
+      sliders.forEach(slider => {
+        const cell = slider.closest('td');
+        const columnIndex = Array.prototype.slice.call(cell?.parentElement?.children).indexOf(cell);
+        const column = sliderColumns.get(columnIndex);
+        if (!column) {
+          sliderColumns.set(columnIndex, [slider]);
+        } else {
+          column.push(slider);
+        }
+      });
+      for (const [columnIndex, sliders] of sliderColumns.entries()) {
+        const header = headerRow.children[columnIndex];
+        const minSliderColumn = () => {
+          sliders.forEach(slider => {
+            const markContainer = _$(slider, 'rc-slider-mark');
+            if (markContainer && !slider.classList.contains('rc-slider-disabled')) {
+              clickElement(markContainer.firstElementChild as HTMLElement);
+            }
+          });
+        };
+        const maxSliderColumn = () => {
+          sliders.forEach(slider => {
+            const markContainer = _$(slider, 'rc-slider-mark');
+            if (markContainer && !slider.classList.contains('rc-slider-disabled')) {
+              clickElement(markContainer.lastElementChild as HTMLElement);
+            }
+          });
+        };
+        const disabled = sliders.every(slider => slider.classList.contains('rc-slider-disabled'));
```

Comment:
Code style for this project: if a lambda has only one parameter, call it `x`. Takes less time to think about a proper name, and reads as clearly as with a full name. Use full names when `x` results in an unclear code.
```suggestion
      const disabled = sliders.every(x => x.classList.contains('rc-slider-disabled'));
```